cromwell.example.backends/AWS.conf

```diff
@@ -9,60 +9,49 @@
 # Documentation
 # https://cromwell.readthedocs.io/en/stable/tutorials/AwsBatch101/
 
-aws {
-
-  application-name = "cromwell"
-
-  auths = [
-    {
-      name = "default"
-      scheme = "default"
-    }
-  ]
-
-  region = "<your-region>"
-}
-
-engine {
-  filesystems {
-    s3 {
-      auth = "default"
+  aws {
+    application-name = "cromwell"
+    auths = [{
+        name = "default"
+        scheme = "default"
+    }]
+    region = "${AWS::Region}"
+  }
+  database {
+    profile = "slick.jdbc.MySQLProfile$"
+    db {
+      driver = "com.mysql.cj.jdbc.Driver"
+      url = "jdbc:mysql://${DBAddress}/cromwell?rewriteBatchedStatements=true&useSSL=false"
+      user = "${DBUsername}"
+      password = "${DBPassword}"
+      connectionTimeout = 5000
     }
   }
-}
-
-backend {
-  default = "AWSBATCH"
-  providers {
-    AWSBATCH {
-      actor-factory = "cromwell.backend.impl.aws.AwsBatchBackendLifecycleActorFactory"
-      config {
-
-        numSubmitAttempts = 6
-        numCreateDefinitionAttempts = 6
-
-        // Base bucket for workflow executions
-        root = "s3://<s3-bucket-name>/cromwell-execution"
-
-        // A reference to an auth defined in the `aws` stanza at the top.  This auth is used to create
-        // Jobs and manipulate auth JSONs.
-        auth = "default"
-
-        default-runtime-attributes {
-          queueArn: "<your arn here>"
-        }
-
-        filesystems {
-          s3 {
-            // A reference to a potentially different auth for manipulating files via engine functions.
-            auth = "default"
+  call-caching {
+    enabled = true
+    invalidate-bad-cache-results = true
+  }
+  engine { filesystems { s3 { auth = "default" } } }
+  backend {
+    default = "AWSBATCH"
+    providers {
+      AWSBATCH {
+        actor-factory = "cromwell.backend.impl.aws.AwsBatchBackendLifecycleActorFactory"
+        config {
+          numSubmitAttempts = 10
+          numCreateDefinitionAttempts = 10
+          root = "s3://${S3BucketName}/cromwell-execution"
+          auth = "default"
+          default-runtime-attributes { queueArn = "${BatchQueue}" , scriptBucketName = "${S3BucketName}" }
```

Comment:
Since this is an example `.conf`, I quite like having the longform commentary alongside the option values.